Automattic/newspack-plugin
# Move components demo into its own wizard

### All Submissions:






### Changes proposed in this Pull Request:



Related #64 

This PR moves the components demo stuff into its own wizard. This will enable development of the Subscriptions Wizard.

### How to test the changes in this Pull Request:

1. `npm ci; npm run clean; npm run build:webpack`
2. Go to Newspack > Components Demo and verify components demo works well.

### Other information:
